### ISSUE TITLE
Review Logging and Trace IBM docs to ensure all relevant info has been migrated

### ISSUE THREAD

**Issue by dmuelle** [OpenLiberty/docs#5041]
https://ibmdocs-test.mybluemix.net/docs/en/SSAW57_liberty_test?topic=liberty-logging-trace

though most of this info has been migrated to https://www.openliberty.io/docs/latest/log-trace-configuration.html, some of the material after table 1 on the IBM Docs page has not been moved. Need to evaluate what is there against what is in OL and OL autogen. Possibly add a new "trace" section to cover the missing info.

**Comment by dmuelle** [comment]
## Peer review

Looks good- a few suggestions

- For the section title, I think it should be "Trace log detail levels", since these settings apply specifically to the trace log and not the console or messages log. And the table should be "Valid log detail levels for the trace log"

- "You can specify the trace specification to selectively enable trace. The logging properties can be set in the server configuration file by adding a logging element to the server configuration file as follows:"

--->

You can specify the `traceSpecification` attribute for the` logging` element in your` server.xml` file to selectively enable trace and set the trace log detail level for specific components in your server configuration. The following example specifies the `audit` detail level for all code that is running on the server and the `finest` detail level for an example package. 

```xml
<logging traceSpecification="*=audit:com.myco.mypackage.*=finest"/>
```

In this example, the two detail level specifications are separated by a colon (`:`). The wildcard value `*=audit` applies to all traceable code that is running on the server. The `:com.myco.mypackage.*=finest` value applies to all classes with the package name that begins with `com.myco.mypackage.`.


- "The following table lists the valid log detail levels that you can specify in the trace specification attribute for application servers at Open Liberty. An empty value is allowed and treated as 'disable all trace'. "

--->

The following table lists the valid log detail levels that you can specify in the `traceSpecification` attribute. You can specify an empty value for any component to disable all trace for that component. 

- Remove "Trace information -" fromn the trhee descriptions where it appears as it doesn't add any value.
- "General trace + method entry, exit, and return values"--> General trace that including method entry, exit, and return values
-  "If you create custom levels, all includes those levels..."--->  If you create custom levels, `all` includes those levels...

**Comment by dmuelle** [comment]
I think this looks good an you can send it to staging

**Comment by ramkumar-k-9286** [comment]
@dmuelle 

File moved to Staging: https://docs-staging-openlibertyio.mybluemix.net/docs/22.0.0.4/log-trace-configuration.html

Regards,
Ramkumar.

**Comment by dmuelle** [comment]
changes on vNExt and will publish with 22.0.0.4

**Comment by ramkumar-k-9286** [comment]
The file has been added to vNext. Will get published in the 22.0.0.4 release.

Regards,
Ramkumar
